lobsterpy/cohp/analyze.py

```diff
@@ -187,11 +187,11 @@ def setup_env(self):
 
         # determine cations and anions
         try:
-            if self.whichbonds == "cation-anion":
+            if self.which_bonds == "cation_anion":

```

Comment:
`cation-anion` is not a parameter. Just a string that we pass.

Reply:
But the string is the thing we change in atomate2 and it was pointed out that is the weird fix? What exactly I should do in atomate2 then? I do not get it 😢

Reply:
Maybe, we don't need the change in atomate2.